packages/diagram/lib/diagram.dart

```diff
@@ -244,6 +255,9 @@ class DiagramController {
   /// files.
   Directory outputDirectory;
 
+  double get pixelRation => _binding.pixelRatio;
```

Comment:
ration?

Reply:
Whoops!  Clearly not actually used yet... Fixed.